src/app/atoms/DonationBanner/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from 'react';
+
+import { Anchor } from 'app/atoms/Anchor';
+import { ReactComponent as Ukraine } from 'app/icons/ukraine.svg';
+import { T } from 'lib/i18n';
+
+import { DonationBannerSelectors } from './selectors';
+
+export const DonationBanner: FC = () => (
+  <Anchor
+    className="flex flex-col items-center justify-center"
+    style={{
+      borderRadius: '4px',
+      maxWidth: '100px',
+      height: '28px',
+      backgroundColor: '#E5F2FF'
+    }}
+    href="https://donate.tezos.org.ua"
+    target="_blank"
+    rel="noopener noreferrer"
```

Comment:
Maybe we need to add `target` and `rel` default values in the `Anchor` component